app/models/concerns/runner_user.rb

```diff
@@ -5,15 +5,13 @@ module RunnerUser
 
   included do
     def s3_bytes_used(since: 100.years.ago)
-      runs.where('created_at > ?', since).sum(0) do |run|
-        begin
-          s3_run_object = $s3_bucket_internal.object("splits/#{run.s3_filename}")
-          s3_run_object.exists? ? s3_run_object.content_length : 0
-        rescue Aws::S3::Errors::Forbidden => e
-          Rollbar.warn(e, 'Missing/forbidden S3 file while counting runs', run_id: run.id, s3_filename: run.s3_filename)
-          0
-        end
-      end
+      runs.where('created_at > ?', since).sum do |run|
+        s3_run_object = $s3_bucket_internal.object("splits/#{run.s3_filename}")
+        s3_run_object.exists? ? s3_run_object.content_length : 0
+      rescue Aws::S3::Errors::Forbidden => e
+        Rollbar.warn(e, 'Missing/forbidden S3 file while counting runs', run_id: run.id, s3_filename: run.s3_filename)
+        0
+      end || 0
```

Comment:
Wow, I knew you could `rescue` without a `begin` at the method-level, but I had no idea you could do it for arbitrary blocks. Sweet!

Reply:
I wanna say you can do it for any kind of block (method, do, if/else) but don't quote me on that.